ansible_base/authentication/models/authenticator_map.py

```diff
@@ -27,6 +27,7 @@ class Meta:
         null=False,
         on_delete=models.CASCADE,
         help_text="The authenticator this mapping belongs to",
+        related_name="authenticator_map",
```

Comment:
It's weird for you to declare a `related_name` that's singular. "authenticator_maps" would make much more sense.

Reply:
Good callout, fixed.